mslib/ogcwms.py

```diff
@@ -54,7 +54,7 @@
 """
```

Comment:
This file has been copied form 
https://pypi.org/project/OWSLib/
version 0.14.
Would this be a good time to rebase to a more recent version? Maybe they support 1.3.0 in the meantime?

Reply:
I will take a look at that.
From what I can see, they do support 1.3.0, but hardcoded in a seperate class from 1.1.1, so it is not backwards compatible out of the box.
Also all the modifications in ogcwms for MSS need to stay. So this is a bit of a fickle thing to rebase.

Reply:
Done